generators/cleanup.js

```diff
@@ -156,4 +156,7 @@ function cleanupOldServerFiles(generator, javaDir, testDir, mainResourceDir, tes
         generator.removeFile(`${testDir}web/rest/ProfileInfoResourceIntTest.java`);
         generator.removeFile('gradle/mapstruct.gradle');
     }
+    if (generator.isJhipsterVersionLessThan('5.2.1')) {
```

Comment:
@vishal423 Is this the correct version to use? `ElasticsearchConfiguration.java` is removed in this PR and no longer needed in 5.2.2+.

Reply:
looks good

Reply:
yes this API is really confusing!! I'm not totally sure, shouldn't that be 5.2.2?

Reply:
That’s sounds more logical. I’ll fix it.

Reply:
yeah, looks like I mis-interpreted the api. I am bit lost in jhipster versioning :confused: